hbase-server/src/main/jamon/org/apache/hadoop/hbase/tmpl/regionserver/BlockCacheTmpl.jamon

```diff
@@ -216,11 +216,30 @@ org.apache.hadoop.util.StringUtils.TraditionalBinaryPrefix;
         <td>Block requests that were cache misses but only requests set to use block cache</td>
     </tr>
     <tr>
-        <td>Hit Ratio</td>
+        <td>All Time Hit Ratio</td>
         <td><% String.format("%,.2f", bc.getStats().getHitRatio() * 100) %><% "%" %></td>
         <td>Hit Count divided by total requests count</td>
     </tr>
-
+    <%for int i=0; i<bc.getStats().getNumPeriodsInWindow(); i++ %>
```

Comment:
so, will this page result in a very large page or unlimited length if the region server for e.g. 7 days? or is the paginated feature enabled automatically ? if not , can we implement it ?

Reply:
Good point. The number of periods is configurable (we set 12 by default), so it could be set to a very high value. I haven't thought about pagination, let me check on this.